js/helper.js

```diff
@@ -48,42 +48,42 @@ function mockBinaryPath(mockPath) {
  * @returns {string[]} An arguments array that can be passed via command line.
  */
 function serializeOptions(schema, options) {
-  return Object.keys(schema).reduce((newOptions, option) => {
+  return Object.keys(schema).reduce((newOptions2, option) => {
     const paramValue = options[option];
     if (paramValue === undefined) {
-      return newOptions;
+      return newOptions2;
```

Comment:
Not sure why I had to rename this, but `npm test` would fail otherwise

Reply:
Because you're otherwise shadowing that name in line 57, which in JavaScript is a common bug and thus linted against.

Reply:
Amazing, I would've expected that thing to make a new scope